packages/twenty-front/src/modules/workflow/search-variables/components/VariableTagInput.tsx

```diff
@@ -130,6 +168,29 @@ export const VariableTagInput = ({
     onUpdate: ({ editor }) => {
       deboucedOnUpdate(editor);
     },
+    editorProps: {
+      handleKeyDown: (view, event) => {
+        if (event.key === 'Enter' && !event.shiftKey) {
+          event.preventDefault();
+
+          const { state } = view;
+          const { tr } = state;
+
+          // Insert hard break using the view's state and dispatch
+          const transaction = tr.replaceSelectionWith(
+            state.schema.nodes.hardBreak.create(),
+          );
+
+          view.dispatch(transaction);
+
+          return true;
+        }
+        return false;
+      },
+    },
+    enableInputRules: false,
+    enablePasteRules: false,
```

Comment:
Does it have something to do with the fact that we can't copy and paste from other `<VariableTagInput />` components and get the variables correctly wired? I'm not saying that we absolutely want that behavior, I'm just wondering what problem you solved by disabling `enabledPasteRules`.


https://github.com/user-attachments/assets/61857248-b93f-42d6-bc28-c3b84c2a60f6

Reply:
I am not sure. Tiptap by default enable a lot of features that we do not need. Following the doc, their recommendation is for exemple to use a `StarterKit` for several extensions. And enabling input and paste rules that I did not fully understand https://tiptap.dev/docs/editor/extensions/custom-extensions/extend-existing#paste-rules. By default I prefer removing these and put back only the ones I need later.

But to answer your video, no, the behavior si the same with or without those paste rules

Reply:
Okay! Great! I agree with you 💯